spock-junit4/src/main/java/org/spockframework/junit4/JUnitDescriptionGenerator.java

```diff
@@ -43,9 +43,6 @@ public static Description describeSpec(SpecInfo spec) {
       if (feature.isExcluded()) {
         continue;
       }
-      if (feature.isReportIterations()) {
```

Comment:
Why is this not needed anymore?

Reply:
Well, the comment says `// don't report up-front because IDEs don't handle this well`.
But no IDE is seeing this anymore, the description is only used for calling the rules.